src/v0/util/index.js

```diff
@@ -1109,6 +1109,28 @@ function getDestinationExternalID(message, type) {
   return destinationExternalId;
 }
 
+/**
+ * Function to get the the externalId array with the given type
+ * @param {*} message {"context":{"externalId":[{"type":"customerListId","id":1},{"type":"customerListId","id":2}]}}
```

Comment:
It is not correct. We should take an array as id.
{"context":{"externalId":[{"type":"customerListId","id":[1,2]}]}}

Reply:
@shrouti1507 suggested this  in Clickup. So, i am using the same here. Let me know if i need to use array here

Reply:
updated